docs/_data/docstoc.yml

```diff
@@ -9,8 +9,8 @@
           url: mdteclipseoverview.html
         - title: 'Installing Codewind for Eclipse'
           url: mdt-eclipse-installinfo.html
-        - title: 'Using a template repository of your own'
-          url: mdteclipseusingadifferenttemplate.html
+        - title: 'Working with templates'
```

Comment:
This part belongs to Codewind for Eclipse, you can let @eharris369 to review.

Reply:
Erin has already reviewed on PR https://github.com/eclipse/codewind-docs/pull/208 - for some reason the changes didn't appear in my local copy of the docs so I added them in again, just to be sure.